src/commands/governance/submit_proposal/motion.rs

```diff
@@ -0,0 +1,59 @@
+use crate::lib::agent::create_waiter;
+use crate::lib::error::NnsCliResult;
+use crate::lib::nns_types::governance::governance_canister_id;
+
+use candid::{CandidType, Decode, Encode};
+use clap::Clap;
+use ic_agent::Agent;
+use ic_nns_common::pb::v1::NeuronId as NeuronIdProto;
+use ic_nns_common::types::NeuronId;
+use ic_nns_governance::pb::v1::manage_neuron_response::Command::{Error, MakeProposal};
+use ic_nns_governance::pb::v1::proposal::Action;
+use ic_nns_governance::pb::v1::{
+    manage_neuron::Command, ManageNeuron, ManageNeuronResponse, Motion, Proposal,
+};
+
+const MANAGE_NEURON_METHOD: &str = "manage_neuron";
+
+/// Submit a motion to the IC.
```

Comment:
s/motion/motionproposal
```suggestion
/// Submit a motion proposal to the IC.
```